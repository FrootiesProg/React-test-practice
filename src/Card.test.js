import React from "react";
import renderer from "react-test-renderer"; 
import { render } from "@testing-library/react";
import Card from "./Card";

test("Card component renders without crashing", () => {
  render(<Card />);
});

test("Card component snapshot", () => {
  const component = renderer.create(<Card />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
