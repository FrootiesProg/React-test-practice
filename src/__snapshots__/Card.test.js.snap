// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component snapshot 1`] = `
<div
  className="Card"
>
  <h4
    className="Card-title"
  />
  <img
    className="Card-image"
  />
  <small
    className="Card-small"
  >
    Image 
     of 
    .
  </small>
</div>
`;
